tf2/src/buffer_core.cpp

```diff
@@ -35,6 +35,7 @@
 #include <string>
 #include <utility>
 #include <vector>
+#include <string.h>
```

Comment:
string.h is a C library, so it's better to use:
```suggestion
#include <cstring>
```

Reply:
You'd also want to place the header inclusion after line 32 to maintain alphabetical ordering of the headers.